pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -545,6 +547,11 @@ func handleLastRebootReason(ctx *nodeagentContext) {
 			ctx.rebootReason)
 		agentlog.DiscardCommonRebootReason()
 	}
+	ctx.panicTrace, ctx.panicTime = agentlog.GetPanicTrace()
+	if ctx.panicTrace != "" {
+		log.Errorf("shutdown at %s:\n %s", ctx.panicTime.String(), ctx.panicTrace)
```

Comment:
I wonder how big this can become. If we have multiple agents panic there will be multiple stacl traces. Plus random broken packages which print to stdout/stderr will also end up there.
So maybe it should be broken up into lines before logging?

Reply:
Line by line splitting makes it very hard to read the stack traces in kibana. Since each stack trace starts with "goroutine" keyword, we could use the same splitting method that we use for Fatal stack trace. Send one complete stack trace per log message.
Does that sound good to you @eriknordmark ?

Reply:
See suggestion about per agent file; that limits the size

Reply:
already did